### DISCUSSION TITLE
How to leave ecochia pool

### DISCUSSION THREAD

**Discussion by MKOSTIOUKOVITCH** [Chia-Network/chia-blockchain#12231] [Support]
ecochia (https://ecoxch.io) pool is no longer functioning. How can I withdraw my NFT plots from this pool. Standard switching to self pooling - does not work. For several days I see the message "Pending" and nothing happens.

**Comment by loppefaaret** [comment]
It should jsut work, if you select to self pool, you are not depending on anything but a transaction from your client, getting through to the network - perhaps after several days waiting, trying to "Delete unconfirmed transactions" in the wallet, and re-trying the self pool is what is needed ?

**Comment by MKOSTIOUKOVITCH** [comment]
Thanks, that's what I tried to do many times. . Did not work out. Maybe there are other options, how to pick up your plots from a non-working pool  ecochia (https://ecoxch.io/)?
